haas/model.py

```diff
@@ -56,6 +56,9 @@ def init_db(create=False, uri=None):
     Session.configure(bind=engine)
     if create:
         get_network_allocator().populate(Session())
+        #x = get_network_allocator()
+        #print x
+        #x.populate(Session())
```

Comment:
deleted the stuff. Moved the 'Console' related logic from Node to ipmi and made requisite stub methods in the OBM superclass